src/Prometheus.Client.MetricPusher.HostedService.Tests/Prometheus.Client.MetricPusher.HostedService.Tests.csproj

```diff
@@ -0,0 +1,32 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+
+    <IsPackable>false</IsPackable>
+    <IsTestProject>true</IsTestProject>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="CleanMoq" Version="1.0.0" />
+    <PackageReference Include="FluentAssertions" Version="6.11.0" />
+    <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="7.0.0" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.5.0" />
+    <PackageReference Include="xunit" Version="2.4.2" />
```

Comment:
Update xunit and xunit.runner.visualstudio to 2.5.0

Reply:
Since xunit.runner.visualstudio v2.5.0 doesn't have compatibility with netcoreapp3.1 so added a conditional reference to v2.4.5 for netcoreapp3.1